contractcourt/htlc_outgoing_contest_resolver_test.go

```diff
@@ -55,7 +55,7 @@ func TestHtlcOutgoingResolverRemoteClaim(t *testing.T) {
 			TxIn: []*wire.TxIn{
 				{
 					Witness: [][]byte{
-						{0}, {1}, {2}, preimage[:],
+						{0}, {1}, {2}, preimage[:], {4},
```

Comment:
Perhaps add another test case in a TDD style so the old code path is still exercised?